app/models/submission.rb

```diff
@@ -0,0 +1,18 @@
+class Submission < ApplicationRecord
+  SUBMISSION_STATUSES = [
+    "Received",
+    "In-progress",
+    "Declined",
+    "Accepted",
+    "Completed",
+    "Withdrawn",
+    "Editable",
+  ]
+  belongs_to :user
+  has_many :submission_packets, dependent: :destroy
+  has_many :poems, through: :submission_packets
+
+  validates_presence_of :title 
+  validates :status, presence: true, inclusion: { in: SUBMISSION_STATUSES }
+  validates_presence_of :submitted_to
```

Comment:
Not realizing `validates` was preferred in all cases. I used it on 16 because I was assigning multiple validations to a single attribute, but used `validates_presence_of` in other places because it seemed more descriptive. Easy fix.